rpcs3/Emu/Memory/vm.cpp

```diff
@@ -870,7 +870,13 @@ namespace vm
 		// Actually unmap memory
 		if (!shm)
 		{
+#if defined(__APPLE__) && defined(ARCH_ARM64)
+			// Hack: we don't have 16K page support yet, so unmapping pages can break
```

Comment:
I added a proper workaround for 4k pages in #12139 hoping this hack can be removed.